src/laws/Laws.jl

```diff
@@ -82,8 +82,11 @@ U_law = LawU(nn_model, params; max_NN = 50.0, prescale_bounds = [bounds_H, bound
 function LawU(
     nn_model::NeuralNetwork,
     params::Sleipnir.Parameters;
-    max_NN::Union{F, Nothing} = 50.0,
-    prescale_bounds::Union{Vector{Tuple{F,F}}, Nothing} = [(0.0, 300.0), (0.0, 0.5)],
+    max_NN::Union{F, Nothing} = nothing,
+    prescale_bounds::Union{Vector{Tuple{F,F}}, Nothing} = nothing,
+    precompute_interpolation::Bool = true,
+    precompute_VJPs::Bool = true,
+    dummy_float::F = 1.0, # This is just to recognize the type F when no float argument is provided. Not always, but sometimes this produces an error of "F not been defined"... I am sure there is a better solution to this but I couldn't find it. Doing this for now.
```

Comment:
I think this will solve your problem
```suggestion
    max_NN::Union{<: AbstractFloat, Nothing} = nothing,
    prescale_bounds::Union{Vector{Tuple{<: AbstractFloat,<: AbstractFloat}}, Nothing} = nothing,
    precompute_interpolation::Bool = true,
    precompute_VJPs::Bool = true,
```

Reply:
Seems to be working for now!!! Thank you!